### PR TITLE
added drivers for the Brother MFC-9130CW

### PR DESCRIPTION
**Pull request by CleverCompiler**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/misc/cups/drivers/mfc9130cwcupswrapper/default.nix [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    chmod +x $out/lib/cups/filter/brother_lpdwrapper_mfc9130cw
+    '';
+
+  meta = with lib; {
+    description = "Brother MFC-9130CW CUPS wrapper driver";
+    homepage = "http://www.brother.com/";
+    sourceProvenance = with sourceTypes; [ binaryNativeCode ];
+    license = licenses.gpl2Plus;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ hexa ];
```
Comment (mweinelt) [inline pkgs/misc/cups/drivers/mfc9130cwcupswrapper/default.nix:68]:
```suggestion
    maintainers = with maintainers; [ ];
```

File: pkgs/misc/cups/drivers/mfc9130cwlpr/default.nix [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        gawk
+      ]}
+  '';
+
+  meta = with lib; {
+    description = "Brother MFC-9130CW LPR printer driver";
+    homepage = "http://www.brother.com/";
+    sourceProvenance = with sourceTypes; [ binaryNativeCode ];
+    license = licenses.unfree;
+    maintainers = with maintainers; [ hexa ];
```
Comment (mweinelt) [inline pkgs/misc/cups/drivers/mfc9130cwlpr/default.nix:73]:
```suggestion
    maintainers = with maintainers; [ ];
```

**Review by mweinelt — COMMENTED** [review]
Sorry, not going to maintain your printer drivers. If you want these merged, create your own maintainer handle and add it to the packages.

File: pkgs/misc/cups/drivers/mfc9130cwcupswrapper/default.nix (hunk 1 above)
Comment (CleverCompiler) [reply pkgs/misc/cups/drivers/mfc9130cwcupswrapper/default.nix]: oops,  Sorry I used your package as a template.  I've updated the maintainer to be me.

**Comment by mweinelt** [comment]
Your first commit should create a maintainer object in `maintainers/maintainer-list.nix`. Then your packages can reference your maintainer object.

**Pull request closed without merging**